tracer/src/Datadog.Trace/DataStreamsMonitoring/DataStreamsManager.cs

```diff
@@ -83,16 +83,12 @@ public async Task DisposeAsync()
     public void InjectPathwayContext<TCarrier>(PathwayContext? context, TCarrier headers)
         where TCarrier : IBinaryHeadersCollection
     {
-        if (!IsEnabled)
+        if (!IsEnabled || context is null)
```

Comment:
If you're going to make this change, you have to remove the `Log.Debug("Attempted to inject null pathway context");` log at the bottom as otherwise you're going to log that with _every_ injection

Reply:
makes sense, removed the debug log